rpc/src/rpc.rs

```
@@ -2587,7 +2589,11 @@ fn get_spl_token_owner_filter(program_id: &Pubkey, filters: &[RpcFilterType]) ->
                         if bytes.len() == PUBKEY_BYTES {
                             owner_key = Pubkey::try_from(bytes).ok();
```

question: if Pubkey::try_from() fails we still silently ignore the filter,
should we error out here too?

---

We could! The line above asserts that there are exactly 32 bytes though, so it's _coincidentally_ impossible to fail `try_from(bytes)`.

¯\\_(°ペ)\_/¯